packages/purpose-template-process/src/services/validators.ts

```diff
@@ -657,3 +657,19 @@ export async function validateEservicesDisassociations(
 
   return validPurposeTemplateResult(validEServiceDescriptorPairs);
 }
+
+export function hasRoleToAccessDraftPurposeTemplates(
```

Comment:
```suggestion
export function hasRoleAccessToDraftPurposeTemplates(
```

Reply:
I may be really tired 😂  but in my mind `hasRoleAccessToDraftPurposeTemplates` means "has the given role access to draft purpose templates"?
I'm not sure about the meaning of `hasRoleToAccessDraftPurposeTemplates`.

But it is a very minor point